build-tests/package-extractor-test-03/package.json

```diff
@@ -5,5 +5,8 @@
   "private": true,
   "scripts": {
     "_phase:build": ""
+  },
+  "devDependencies": {
+    "@types/node": "ts3.9"
```

Comment:
Curious... I've never seen this version specifier format. How does this work? Why is this needed?

Reply:
I saw them in `common/config/rush/common-versions.json`, and the comment says these are versions are used by testing projects.